Documentation/Getting-Started/glossary.md

```diff
@@ -74,6 +74,10 @@ A [host storage cluster](../CRDs/Cluster/host-cluster.md) is where Rook configur
 
 The [Rook Krew plugin](../Troubleshooting/krew-plugin.md) is a tool to help troubleshoot your Rook cluster.
 
+### Koor Storage Distribution (KSD)
```

Comment:
Why is KSD in the `Rook` section of the glossary? I think it should be in a separate section

Reply:
@galexrt ah okay, I'll create another Section for `Koor`